src/code/PublishPSResource.cs

```diff
@@ -232,9 +232,20 @@ protected override void EndProcessing()
                 }
 
                 // validate that the module manifest has correct data
-                if (!IsValidModuleManifest(resourceFilePath))
+                string[] errorMsgs = null;
+                try
                 {
-                    return;
+                    Utils.ValidateModuleManifest(resourceFilePath, out errorMsgs);

```

Comment:
Declare inline if we can
```suggestion
                    Utils.ValidateModuleManifest(resourceFilePath, out string[] errorMsgs);

```

Reply:
I accepted these changes but reverted it because the finally block is using the errorMsgs variable so it needs to be declared before the try block.